### PR TITLE
4919 expected missings

### PR DESCRIPTION
**Pull request by MirijaHa**
Very low prio:
Unfortunately pattern for production_plan_company_technology does not seem to match. I looked sharply at the ticket repeatedly but cannot spot the mistakes, @jacobvjk  maybe you have more look.

### CODE REVIEW

File: R/check_data.R [hunk 1]
```diff
@@ -93,3 +93,30 @@ check_company_ticker_mapping <- function(data) {
...
+#' On certain variables missings appear that are not problematic but are
+#' expected. Here it is checked whether those missings adher to expected
+#' patterns.
+#'
+#' @param data Data that expected missings are checked on.
+#'
+#' @return Returns input invisibly.
+check_expected_missings <- function(data) {
+  n_missings_production_plan_company_technology <- sum(is.na(data$production_plan_company_technology))
+  expected_n_missings_production_plan_company_technology <- nrow(data) * (length(unique(data$year)) - length(seq(min(data$year), min(data$year) + time_horizon_lookup)) / length(unique(data$year)))
```
Comment (jacobvjk) [inline R/check_data.R:108]: @MirijaHa I missed one set of brackets. This line needs to be: `expected_n_missings_production_plan_company_technology <- nrow(data) * ((length(unique(data$year)) - length(seq(min(data$year), min(data$year) + time_horizon_lookup))) / length(unique(data$year)))`

**Review by jacobvjk — CHANGES_REQUESTED** [review]
See comment. Running this version seems to work for the loan book example

**Comment by MirijaHa** [comment]
@jacobvjk changed the bracket and simplified the patterns a bit, runs through smoothly now

**Review by jacobvjk — APPROVED** [review]
lgtm

**Pull request merged**